src/snowflake/connector/ocsp_snowflake.py

```diff
@@ -1083,7 +1099,7 @@ def validate_by_direct_connection(self, issuer, subject, telemetry_data, hostnam
                 logger.debug("using OCSP response cache")
 
             if not ocsp_response:
-                logger.debug('No OCSP URL is found.')
+                telemetry_data.set_event_sub_type('OCSPResponseUnavailable')
```

Comment:
can you make the sub type constant variables so that the application can refer the same value if required?

Reply:
Sure that makes sense.